src/database/UIDatabase.ts

```diff
@@ -17,7 +17,56 @@ import { version as appVersion } from '../../package.json';
 
 const { THIS_STORE_NAME_ID, APP_VERSION } = SETTINGS_KEYS;
 
-const translateToCoreDatabaseType = type => {
+export type RealmDatabaseType =
```

Comment:
this should be `| CoreDatabaseType` also actually. It's a superset of all the types. 

So, all the core database schema types are of type `CoreDatabaseType` and the additional types i.e. `Vaccine`, which is a filtered subset of `Item` , are the additions for `RealmDatabaseType`.

Currently, if I use `UIDatabase.objects("Transaction")`, which is valid and common, the key is not valid.